src/js/server/devpmt-express.js

```diff
@@ -0,0 +1,537 @@
+                        args: ["%prefsServerURL/preferences", {
+                            prefsServerURL: "{gpii.devpmt}.options.prefsServerURL"
+                        }]
+                    }
+                },
+                writable: true,
+                writeMethod: "POST"
+            }
+        },
+        prefsSafeKeyCreationDataSource: {
+            type: "kettle.dataSource.URL",
+            options: {
+                url: {
+                    expander: {
+                        funcName: "fluid.stringTemplate",
+                        args: ["%prefsServerURL/prefssafe-key-create", {
+                            prefsServerURL: "{gpii.devpmt}.options.prefsServerURL"
+                        }]
+                    }
+                },
+                writable: true,
+                writeMethod: "POST"
+            }
+        },
+        prefsSafeByGpiiKeyDataSource: {
+            type: "kettle.dataSource.URL",
+            options: {
+                url: {
+                    expander: {
+                        funcName: "fluid.stringTemplate",
+                        args: ["%prefsServerURL/prefssafe/%prefsSafeId", {
+                            prefsServerURL: "{gpii.devpmt}.options.prefsServerURL"
+                        }]
+                    }
+                },
+                termMap: {
+                    prefsSafeId: "%prefsSafeId"
+                }
+            }
+        },
+        cloudSafeCredCreateDataSource: {
+            type: "kettle.dataSource.URL",
+            options: {
+                url: {
+                    expander: {
+                        funcName: "fluid.stringTemplate",
+                        args: ["%prefsServerURL/add-cloud-credentials/%prefsSafeId", {
+                            prefsServerURL: "{gpii.devpmt}.options.prefsServerURL"
+                        }]
+                    }
+                },
+                termMap: {
+                    prefsSafeId: "%prefsSafeId"
+                },
+                writable: true,
+                writeMethod: "PUT"
+            }
+        },
+        cloudSafeUnlockDataSource: {
+            type: "kettle.dataSource.URL",
+            options: {
+                url: {
+                    expander: {
+                        funcName: "fluid.stringTemplate",
+                        args: ["%prefsServerURL/unlock-cloud-safe", {
+                            prefsServerURL: "{gpii.devpmt}.options.prefsServerURL"
+                        }]
+                    }
+                },
+                writable: true,
+                writeMethod: "POST"
+            }
+        }
+    },
+    invokers: {
+        stop: {
+            func: "{that}.destroy"
+            // funcName: "gpii.express.stopServer",
+            // args: ["that"]
+        }
+    }
+});
+
+gpii.devpmt.initialize = function (that) {
+    var personaKeys = ["alice", "davey", "david", "elmer", "elod", "livia"];
```

Comment:
Do you think if it makes sense to move the definition of `personaKeys` and `osList` at line 519 to be configurable options of `gpii.devpmt` component?

Reply:
It's not a bad idea. I completely removed personakeys since it's no longer used, and then moved osList to the `gpii.devpmt` options.